config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+# config/initializers/sidekiq.rb
+
+Sidekiq.configure_server do |config|
+  config.redis = { url: 'redis://localhost:6379/0', namespace: "wbooks_#{Rails.env}" }
```

Comment:
the REDIS url should be handled via ENV variable, when you deploy to some cloud service it's not always going to be localhost